### ISSUE TITLE
Corporation Industry\Wallet

### ISSUE THREAD

**Issue by Junelicious** [Ionaru/EVIE#61]
Awesome tool, though I think it would be much, much better if it also supported corporation-wide industry jobs and wallet history. 

Ideally, I'd want it to have tabs for personal\corp on top of existing sections
Alternatively, a corp icon with tabs for industry\wallet\maybe other stuff (corp killmail display spring to mind as a relatively easy and useful feature) would be amazing to help manage the corp.

**Comment by Ionaru** [comment]
Good idea! I think a corporation page with all kinds of tabs with info would be very useful indeed.
